homework-submission/myapp/src/App.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { BrowserRouter, Route, MemoryRouter, Link, Switch } from 'react-router-dom';
+
+import StaticList from './Components/Week1/App';
+import DynamicList from './Components/Week2/App';
+import APICall from './Components/Week3/App';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <MemoryRouter>
+          <ul>
```

Comment:
Why are you using `MemoryRouter`?!!

Reply:
if I did not put it I git this error(Error: Invariant failed: You should not use <Link> outside a <Router>)

Reply:
As i said in the comment BrowserRouter should wrap your whole app which i gave you two solutions

Link is a component which basically contains `a` tag